src/main/java/com/github/dockerjava/api/model/PushResponseItem.java

```diff
@@ -18,7 +18,7 @@
      */
     @JsonIgnore
     public boolean isErrorIndicated() {
-        if (getError() == null)
+        if (getError() == null && getErrorDetail() == null)
```

Comment:
I think this method should be pushed up to `ResponseItem` class. Then it should be used in `BuildResponseItem.isBuildSuccessIndicated()` and `PullResponseItem.isPullSuccessIndicated()` as pre-condition check.